### DISCUSSION TITLE
AR QuickLook banner with parameters when auto generating USDZ blob

### DISCUSSION THREAD

**Discussion by Jamedie** [google/model-viewer#4575] [Q&A]
### Description

After searching and digging through older issues, I haven't been able to use a blob with a banner. For now, only the 'allowsContentScaling' parameter works.

The following live demo, which includes multiple QuickLook parameters, does not work on my iPhone XR and iPhone 13
https://modelviewer-auto-usdz-params.glitch.me/

### Browser Affected
- [X ] Safari

### OS
- [X ] iOS 17.1.1

### AR
- [X ] QuickLook

**Comment by elalish** [comment]
Is this the same as #4570?

**Reply by Jamedie** [reply]
In this way, I can see he use a direct link for open AR quick look with custom query, but I want to generate it.

In this link https://modelviewer-auto-usdz-params.glitch.me/

The focused example is:

Multiple QuickLook params + ModelViewer Fixed Scaling

/Astronaut.glb#applePayButtonType=plain✓outTitle=Spaceman%20Toy✓outSubtitle=1969 Limited Edition≺ice=$99&canonicalWebPageURL=https%3A%2F%2Fdeveloper.apple.com

I found the link at : 
https://github.com/google/model-viewer/pull/3434

But in reality, only 'allowsContentScaling' and 
  "canonicalWebPageURL" works; all other parameters are ignored.

**Reply by elalish** [reply]
Your glitch is very helpful - would you mind adding some notes to each example saying what is or isn't working and what is expected?

**Reply by Jamedie** [reply]
Hello, I have created a custom glitch because the linked glitch is not mine, as seen here: https://bubbly-sneaky-swim.glitch.me/. In this project, I have built two basic model viewers. The first version works because it uses an iOS source with a hash to display a banner. However, the second version does not work without a specified iOS source. It exports a USDZ blob while retaining the banner hash, but this does not appear in Quick Look

You can view the source code; if you have any trouble accessing it, I am available to help

**Reply by elalish** [reply]
@milesgreen @hybridherbst Have you run into this problem? Unfortunately I'm remote right now and don't have access to an iOS device for testing.
